docs/user-guide/concepts/data-types/overview.md

```diff
@@ -18,6 +18,7 @@ from Arrow, with the exception of `String` (this is actually `LargeUtf8`), `Cate
 |          | `Float64`     | 64-bit floating point.                                                                                                               |
 | Nested   | `Struct`      | A struct array is represented as a `Vec<Series>` and is useful to pack multiple/heterogeneous values in a single column.             |
 |          | `List`        | A list array contains a child array containing the list values and an offset array. (this is actually Arrow `LargeList` internally). |
+|          | `Array`       | A fixed-length homogeneous array.                                                                                                    |
```

Comment:
Can you name this a `multidimensional array`? Homogenous is superfluous as we all our data containers are.

Reply:
> A fixed-size multidimensional array.